harness/determined/searcher/_search_runner.py

```diff
@@ -191,11 +196,24 @@ def post_operations(
             searcherOperations=[op._to_searcher_operation() for op in operations],
             triggeredByEvent=event,
         )
-        bindings.post_PostSearcherOperations(
-            session,
-            body=body,
-            experimentId=experiment_id,
-        )
+        # TODO: Remove this temporary hack in favor of an actual solution around the issue described
+        # in https://determined-ai.slack.com/archives/C04645NHSP6/p1680201334658789
```

Comment:
can you explain what you are working around instead of leaving a slack hyperlink in the source code?

Reply:
Yep, my bad.

Reply:
Removed and explained in #6988